flink-connector-aws/flink-connector-sqs/archunit-violations/stored.rules

```diff
@@ -0,0 +1,4 @@
+#
+#Tue Feb 22 12:19:26 CET 2022
+Tests\ inheriting\ from\ AbstractTestBase\ should\ have\ name\ ending\ with\ ITCase=54da9a7d-14d2-4632-a045-1dd8fc665c8f
```

Comment:
I am not sure if these auto-generated files should be added to PR?

Reply:
yes this is part of arch tests build